npdarrington/Whats-Cookin
# Fetch functionality

What’s this PR do?  
This PR adds initial fetch functionality to the project; it grabs data from the server and stores it in an object. 

Where should the reviewer start?  
In the newly created fetch.js file. 

How should this be manually tested?  
I am not sure how to manually test this, to be honest. Probably could be tested on the DOM though by running the fetch and checking to see if the data is accessible. 

Any background context you want to provide?  

What are the relevant tickets?  
This closes ticket 44

Screenshots (if appropriate)  

Questions:
